source/API_Reference/SMTP_API/index.html

```diff
@@ -105,7 +105,7 @@
 {% codeblock lang:json %}{
   "to": [
     "<ben@example.com>",
-    "Joe Smith <joe@example.com>"
+    "Joe Smith <joeexampexample@example.com>"
```

Comment:
example@example.com
example@example.com

Reply:
I  have address this issue in my [commit](https://github.com/sendgrid/docs/pull/3206/commits/85181c5f86aa2015a66aecebdd1f6eeaa5b74224)